package.json

```diff
@@ -1,34 +1,63 @@
+    "archiver": "^5.3.0",
+    "chalk": "^4.1.2",
+    "chokidar": "^3.0.1",
+    "cli-cursor": "^3.1.0",
+    "cli-progress": "^3.11.2",
+    "express": "^4.17.1",
+    "findup-sync": "^4.0.0",
+    "fs-extra": "^8.1.0",
+    "inquirer": "8.2.0",
+    "jest": "^29.5.0",
+    "js-yaml": "^4.1.0",
+    "moment": "^2.29.1",
+    "open": "^7.0.3",
+    "ora": "^4.0.3",
+    "p-queue": "^6.0.2",
+    "strip-ansi": "^5.2.0",
+    "table": "^6.6.0",
+    "tmp": "^0.2.1",
+    "update-notifier": "^5.1.0",
+    "yargs": "15.4.1"
+  },
   "devDependencies": {
-    "depcheck": "1.3.1",
+    "axios": "^1.7.2",
+    "mock-stdin": "^1.0.0",
     "eslint": "^8.56.0",
     "husky": "^1.3.1",
-    "lerna": "^6.6.1",
     "lint-staged": "^10.5.4",
     "madge": "^6.1.0",
-    "prettier": "^1.19.1",
-    "sass": "~1.29.0"
+    "prettier": "^1.19.1"
   },
-  "workspaces": [
-    "packages/*",
-    "acceptance-tests"
-  ],
   "scripts": {
-    "check-deps": "node bin/depcheck.js",
-    "lint": "eslint . && prettier --list-different packages/**/*.{js,json}",
-    "prettier:write": "prettier --write packages/**/*.{js,json}",
-    "publish-release": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish --conventional-graduate --exact || echo 'Error: New release can only be published on main branch' >&2",
-    "publish-prerelease": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish prerelease --preid beta --dist-tag next --exact || echo 'Error: New release can only be published on main branch' >&2",
+    "lint": "eslint . && prettier --list-different ./**/*.{js,json}",
+    "prettier:write": "prettier --write ./**/*.{js,json}",
     "test": "jest",
     "test-cli": "cd acceptance-tests && yarn test-headless --stream",
     "test-docker": "yarn build-docker && docker container run -it --rm --name=hs-cli-container hs-cli-image yarn test",
     "build-docker": "docker image build --tag hs-cli-image . && docker image prune -f",
-    "circular-deps": "yarn madge --circular packages"
+    "circular-deps": "yarn madge --circular .",
+    "check-main": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] || (echo 'Error: New release can only be published on main branch' && exit 1)",
+    "pub": "npm publish --tag latest",
+    "push": "git push --atomic origin main v$npm_package_version",
+    "release:major": "yarn check-main && yarn version --major && yarn pub && yarn push",
+    "release:minor": "yarn check-main && yarn version --minor && yarn pub && yarn push",
+    "release:patch": "yarn check-main && yarn version --patch && yarn pub && yarn push",
+    "prerelease:major": "yarn check-main && yarn version --premajor --preeid=beta && yarn pub && yarn push",
+    "prerelease:minor": "yarn check-main && yarn version --preminor --preeid=beta && yarn pub && yarn push",
+    "prerelease:patch": "yarn check-main && yarn version --prepatch --preeid=beta && yarn pub && yarn push",
```

Comment:
These are our new release scripts. They match the ones in local-dev-lib but add 3 new ones to do prereleases